### PR TITLE
home_work3_Login_page

### PR DESCRIPTION
**Pull request by Lytvynenko18**
home_work_3-forms
added pages: Login, "Success"
2aa8191896cb96ccc6b04e19dad11cf5661541c7

### CODE REVIEW

**Comment by Lytvynenko18** [comment]
[e44bff3](https://github.com/MastersAcademy/fe-course-basic-2023/pull/211/commits/e44bff3a23021a91fe82fbb0d9bd483ce3a6efa5) fixed issue with about me section height

**Comment by Lytvynenko18** [comment]
[df2affa](https://github.com/MastersAcademy/fe-course-basic-2023/pull/211/commits/df2affa8943dd80a250d8a36e763138356e550b6) fixed buttom name on the success page

File: homeworks/roman.lytvynenko_Lytvynenko18/GamesProject/login.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+                <label class="login__fields">Password
+                </label>
+                <br>
+                <input class="login__input" type="password" name="auth_pass" placeholder="Enter your email address" required>
+                <br>
+                <label>
+                    <input type="checkbox" name="remember" required> I'm not a robot
+                </label>
+                <br>
+                <button class="login__button" type="submit" name="Login">Login </button>
```
Comment (vladyslav-yermolin-moc) [inline homeworks/roman.lytvynenko_Lytvynenko18/GamesProject/login.html:41]: Please correct the border of the button
Also, your width of the input is fixed to 575px. But by [default](https://developer.mozilla.org/en-US/docs/Web/CSS/box-sizing#content-box), the width of the border is added to this width, and as a result, you have 577px. So the inputs are wider than the button. 
Please fix the width or correct `box-sizing` to [border-box](https://developer.mozilla.org/en-US/docs/Web/CSS/box-sizing#border-box)

 image

**Review by vladyslav-yermolin-moc — CHANGES_REQUESTED** [review]

File: homeworks/roman.lytvynenko_Lytvynenko18/GamesProject/login.html [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+                <a class="header__link" href="./login.html" title="Login">Login</a>
+            </div>
+        </header>
+        <div class="login">
+            <h1 class="login__h1">Login page</h1>
+            <form class="login__form" action="./success.html">
+                <label class="login__fields">Email
+                </label>
+                <br>
+                <input class="login__input" type="email" name="auth_email" placeholder="Enter your email address" required>
```
[the remark this answers is not in the document]
Comment (Lytvynenko18) [reply homeworks/roman.lytvynenko_Lytvynenko18/GamesProject/login.html:29]: fixed in 1ffea05f871d93574677456cc36ef53bd6c48564

File: homeworks/roman.lytvynenko_Lytvynenko18/GamesProject/login.html [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+                <br>
+                <label class="login__fields">Password
+                </label>
+                <br>
+                <input class="login__input" type="password" name="auth_pass" placeholder="Enter your email address" required>
+                <br>
+                <label>
+                    <input type="checkbox" name="remember" required> I'm not a robot
+                </label>
+                <br>
```
[the remark this answers is not in the document]
Comment (Lytvynenko18) [reply homeworks/roman.lytvynenko_Lytvynenko18/GamesProject/login.html]: fixed in 1ffea05f871d93574677456cc36ef53bd6c48564

File: homeworks/roman.lytvynenko_Lytvynenko18/GamesProject/login.html (hunk 1 above)
Comment (Lytvynenko18) [reply homeworks/roman.lytvynenko_Lytvynenko18/GamesProject/login.html:36]: fixed in 1ffea05f871d93574677456cc36ef53bd6c48564

File: homeworks/roman.lytvynenko_Lytvynenko18/GamesProject/login.html [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+                <a class="header__link" href="./index.html" title="About">About</a>
+            </div>
+            <div class="about-me__header__menu">
+                <a class="header__link" href="./login.html" title="Login">Login</a>
+            </div>
+        </header>
+        <div class="login">
+            <h1 class="login__h1">Login page</h1>
+            <form class="login__form" action="./success.html">
+                <label class="login__fields">Email
```
[the remark this answers is not in the document]
Comment (Lytvynenko18) [reply homeworks/roman.lytvynenko_Lytvynenko18/GamesProject/login.html:27]: Could U give more context pls? I used <label><input></label> and see focus (bold frame) on the input field when I click on it. Am I missind somthing?

**Review by vladyslav-yermolin-moc — APPROVED** [review]

File: homeworks/roman.lytvynenko_Lytvynenko18/GamesProject/login.html (hunk 4 above)
[the remark this answers is not in the document]
Comment (vladyslav-yermolin-moc) [reply homeworks/roman.lytvynenko_Lytvynenko18/GamesProject/login.html:27]: Sorry, I had to reply on it.
Based on your code of that moment, you haven't had a connection between `label` and `input` ( no `for`, no wrapping  `<label><input></label>`). And that is why I've made this comment. 
Probably you've added it after.
image

[the remark this answers is not in the document]
Comment (vladyslav-yermolin-moc) [reply homeworks/roman.lytvynenko_Lytvynenko18/GamesProject/login.html:27]: Ou, I've missed it, and resolve this comment. Here I've made an screencast with description

https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/7a9821bf-4df7-4ce4-8a1c-a21cea83a8d5

**Pull request merged**